.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
delete this file please. no .gitignore in any Extensions because no build process to remove them from distributed releases/zips